README.md

```diff
@@ -1,33 +1,101 @@
+The Hub is built on open source software and data standards developed by the
+[Hubverse](https://hubverse.io/). We welcome nowcast submissions from all modelers.
 
-Collaborative and open forecast hubs have emerged as a valuable way to centralize and coordinate predictive modeling efforts for public health. In realms where multiple teams are tackling the same problem using different data inputs and/or modeling methodologies, a hub can standardize targets in ways that facilitate model comparison and the integration of outputs from multiple models into public health practice. This hub uses the open-source architecture and data standards developed by the [hubverse](https://hubverse.io/).
+**Submissions accepted** every Wednesday by 8pm ET, starting October 9, 2024.
 
-While SARS-CoV-2 variant dynamics received most attention from the scientific community in 2021 and 2022, SARS-CoV-2 genomic sequences continue to be generated, and trends in variant frequencies will continue to impact transmission across the US and the world. From a modeling perspective, there is less consensus about a standard way to represent model outputs for multivariate variant frequency predictions than there is for other outcomes. Therefore, a key reason for building and launching this nowcast hub is to help learn about the right way to evaluate and communicate variant dynamics in a collaborative modeling effort, potentially not just for SARS-CoV-2 but also for other rapidly evolving pathogens.
+## Modelers guide to using this hub
+
+This section provides a high-level getting started guide for modelers who want
+to submit the United States SARS-CoV-2 Variant Nowcast Hub.
+
+- See [Background](#background) for details about what modelers will be asked
+to predict and how the hub will evaluation submissions.
+- See the [model-output `README`](model-output/README.md) more details about
+the submission process, including an example file.
+
+### Repository structure
+
+Hubervse-based modeling hubs have the following directory structure. Of these, only `model-metadata`,
+`hub-config`, and `model-output` are relevant to modelers.
+
+```script
+variant-nowcast-hub/
+├─ auxiliary-data/
+├─ hub-config/
+│  ├─ admin.json
+│  ├─ model-metadata-schema.json
+│  ├─ tasks.json      <-- round details, including which clades to model
+│  ├─ validations.yml
+├─ model-metadata/    <-- modelers create metadata files here
+├─ model-output/      <-- modelers submit nowcasts here
+├─ src/
+```
```

Comment:
rearranging the list to match the visual structure of the tree, removing the script tag, and adding the descriptions in text (for accessibility).

````suggestion
Hubervse-based modeling hubs have the following directory structure. Of these, only 
`hub-config/tasks.json`, `model-metadata`, and `model-output` are relevant to modelers.

```
variant-nowcast-hub/
├─ auxiliary-data/
├─ hub-config/
│  ├─ admin.json
│  ├─ model-metadata-schema.json
│  ├─ tasks.json      <----- 1
│  ├─ validations.yml
├─ model-metadata/    <----- 2
├─ model-output/      <----- 3
├─ src/
```

 1. `hub-config/tasks.json` contains round details, including which clades to model
 2. modelers submit model metadata in `model-metadata/`
 3. modelers submit nowcast in `model-output/`
````